content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
+- Calculation of percentile aggregations
+- Customization of tagging
 
-Discover how to submit rate metrics:
+If you send `X` values for a DISTRIBUTION metric `<METRIC_NAME>` in a given time interval, the following aggregations are available for query by default:
 
-* [With a custom Agent Check][1]
-* [With the Datadog API][2]
+| Aggregation           | Description                                                                                                                                          | Datadog In-app Type |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- |
+| `avg:<METRIC_NAME>`   | Represents the average of those `X` values in the time interval                                                                                      | GAUGE               |
+| `count:<METRIC_NAME>` | Represents the number of points sampled in the time interval, `X`. The Agent then sends it as a RATE so it would show in app the value `X/interval`. | RATE                |
+| `max:<METRIC_NAME>`   | Represents the maximum value of those `X` values sent in the time interval.                                                                          | GAUGE               |
+| `min:<METRIC_NAME>`   | Represents the minimum value of those `X` sent in the time interval.                                                                                 | GAUGE               |
+| `sum:<METRIC_NAME>`   | Represents the sum of all `X` values sent in the time interval                                                                                       | GAUGE               |
 
-[1]: /developers/metrics/agent_metrics_submission/?tab=rate
-[2]: /api/?lang=python#post-timeseries-points
 {{% /tab %}}
-{{% tab "GAUGE" %}}
-
-**The `GAUGE` metric submission type represents a value of a particular thing reporting continuously over time.** It is a snapshot of the last recorded value of a particular thing during a defined time interval (flush interval). A `GAUGE` can be used to represent the temperature or memory usage.
+{{< /tabs >}}
 
-For instance, say that the `number.of.requests.gauge` metric is reported every 30 seconds to Datadog with the `GAUGE` type in `server:web_1`.
+### Example
 
-Each value/data point represents the total number of requests received at a point in time. The metric then reports the following values:
+{{< tabs >}}
+{{% tab "COUNT" %}}
 
-* `10` for the first 30 seconds
-* `30` for the second interval of 30 seconds (10 + 20 requests)
-* `30` for the last interval of 30 seconds (no new request are received so the value stays the same)
+Suppose you’re submitting a COUNT metric , `activeusers.basket_size`, from a single host running the Datadog Agent. This metric emits the following values in a 10s time interval: `[1,1,1,2,2,2,3,3]`.
```

Comment:
Suppose you’re submitting a COUNT metric , `activeusers.basket_size`, from a single host running the Datadog Agent. This metric emits the following values in a flush time interval: `[1,1,1,2,2,2,3,3]`.